src/components/NewForm/index.js

```diff
@@ -119,110 +86,85 @@ function NewForm() {
               minWidth: "70%",
             }}
           >
-            <Card style={{ maxWidth: "100%", alignContent: "left" }}>
-              <Form
-                {...formItemLayout}
-                labelWrap
-                onFinish={handleSaveForm}
-                form={settingsForm}
-                onFinishFailed={onFinishFailed}
-              >
-                <Title level={2}>New Form</Title>
-                <Form.Item
-                  name="name"
-                  label="Name of the form"
-                  rules={[{ required: true }]}
-                >
-                  {" "}
-                  <Input onChange={handleNameChange} />{" "}
-                </Form.Item>
-                <Form.Item name="description" label="Enter form description">
-                  {" "}
-                  <Input onChange={handleDescriptionChange} />{" "}
-                </Form.Item>
-                <Form.Item
-                  name="selfSign"
-                  label="Non-anonymous form filling: Ask users to self sign their submissions"
+            <Card
+              style={{ maxWidth: "100%", alignContent: "left" }}
+              tabList={tabList}
+              activeTabKey={activeTab}
+              onTabChange={handleTabChange}
+              title={settingsForm.getFieldValue("name") || "New Form"}
+              extra={
```

Comment:
you can create a separate component/function for this extra, as it has lot of conditional code and its increasing file/jsx size

Reply:
it's just a button, will keep it as is for now